.github/workflows/dapr-perf-components.yml

```diff
@@ -222,7 +222,7 @@ jobs:
         id: setup-go
         uses: actions/setup-go@v5
         with:
-          go-version-file: 'go.mod'
+          go-version: '1.21.9'
```

Comment:
Maybe revert these changes?

Reply:
I'm sorry, I had corrected these, but pushed to the wrong branch. Fixed now.